setup.cfg

```diff
@@ -0,0 +1,89 @@
+[metadata]
+author = Christian Theune
+author_email = ct@flyingcircus.io
+classifiers =
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+description = Mirroring tool that implements the client (mirror) side of PEP 381
+long_description = file:README.md
+long_description_content_type = text/markdown
+license = Academic Free License, version 3
+license_file = LICENSE
+name = bandersnatch
+url = https://github.com/pypa/bandersnatch/
+version = 3.4.0
+
+[options]
+include_package_data = True
+install_requires =
+    aiohttp
+    filelock
+    packaging
+    requests
+    setuptools>40.0.0
+    xmlrpc2
+package_dir =
+    =src
+packages = find:
+python_requires = >=3.6
+
+[options.packages.find]
+where=src
+
+[options.entry_points]
+# This entrypoint group must match the value of bandersnatch.filter.PROJECT_PLUGIN_RESOURCE
+bandersnatch_filter_plugins.v2.project =
+    blacklist_project = bandersnatch_filter_plugins.blacklist_name:BlacklistProject
+    whitelist_project = bandersnatch_filter_plugins.whitelist_name:WhitelistProject
+    regex_project = bandersnatch_filter_plugins.regex_name:RegexProjectFilter
+
+# This entrypoint group must match the value of bandersnatch.filter.RELEASE_PLUGIN_RESOURCE
+bandersnatch_filter_plugins.v2.release =
+    blacklist_release = bandersnatch_filter_plugins.blacklist_name:BlacklistRelease
+    prerelease_release = bandersnatch_filter_plugins.prerelease_name:PreReleaseFilter
+    regex_release = bandersnatch_filter_plugins.regex_name:RegexReleaseFilter
+    latest_release = bandersnatch_filter_plugins.latest_name:LatestReleaseFilter
+    exclude_platform = bandersnatch_filter_plugins.filename_name:ExcludePlatformFilter
+
+console_scripts =
+    bandersnatch = bandersnatch.main:main
+
+zc.buildout =
```

Comment:
We can prob get ird of all the zc + zest config. No one active uses it.

Reply:
I have been tempted to do that many times.

Reply:
I've removed it